apis/nucleus/src/components/listbox/hooks/useExistingModel.jsx

```diff
@@ -6,12 +6,25 @@ export default function useExistingModel({ app, qId, options = {} }) {
   const [modelStore] = useModelStore();
   const { sessionModel } = options;
 
-  if (options.dense) {
-    throw new Error('Option "dense" is not applicable for existing objects.');
-  }
+  const invalidOptions = {
+    dense: options.dense,
+    frequencyMode: options.frequencyMode,
+    checkboxes: options.checkboxes,
+    histogram: options.histogram,
+    title: options.title,
+    stateName: options.stateName,
+    listLayout: options.listLayout,
+  };
+
+  let usedInvalidOption = null;
+  Object.entries(invalidOptions).forEach(([key, value]) => {
+    if (value) {
+      usedInvalidOption = key;
+    }
+  });
```

Comment:
So it will only consider the options invalid if its value is `truthy`? An option with value `false` will pass as ok.

I think something like this could work:
```
const forbiddenOptions = ['dense', 'frequencyMode', 'checkboxes'…];
const usedOptions = Object.keys(options);

// get intersection of arrs
const invalidOptions = usedOptions.filter(Set.prototype.has, new Set(forbiddenOptions));

throw new Error(`Options ${invalidOptions.join(', ')} are invalid…`);

```

Reply:
`usedOptions.filter(Set.prototype.has, new Set(forbiddenOptions));` might be an elegant solution, but it's not very easy to read and understand IMO...

Reply:
Yes, it is good to wrap that in a self-explanatory variable. Could even add a comment about it.